docs/pages/setup/reference/networking.mdx

```diff
@@ -47,26 +49,17 @@ following use cases:
 ## HTTP CONNECT proxies
 
 Some networks funnel all connections through a proxy server where they can be
-audited and access control rules can be applied. For these scenarios, Teleport
-supports HTTP CONNECT tunneling.
+audited and access control rules are applied. For these scenarios, Teleport supports HTTP CONNECT tunneling.
 
-To use HTTP CONNECT tunneling, simply set either the `HTTPS_PROXY` or
-`HTTP_PROXY` variables of the `teleport` daemon's shell environment. When the
-daemon builds and establishes a reverse tunnel to the main cluster, it will
-funnel all traffic through the selected proxy. 
-
-If using the default configuration, Teleport will tunnel ports `3024` (SSH,
-reverse tunnel) and `3080` (HTTPS, establishing trust) through the proxy.
+To use HTTP CONNECT tunneling, simply set either the `HTTPS_PROXY` or `HTTP_PROXY` environment variables and when Teleport builds and establishes the reverse tunnel to the main cluster, it will funnel all traffic through the proxy. Specifically, if using the default configuration, Teleport will tunnel ports `3024` (SSH, reverse tunnel) and `3080` (HTTPS, establishing trust) through the proxy.
```

Comment:
Minor, but I would remove the word 'simply'.  Most people reading this will know how to set environment variables but a few might not.  Perhaps include an example like `export HTTPS_PROXY=https://my.proxy:8080`.